@xen-orchestra/lite/src/components/PowerStateIcon.vue

```diff
@@ -11,24 +11,20 @@ import {
   faQuestion,
   faStop,
 } from "@fortawesome/pro-solid-svg-icons";
-import FormWidget from "@/components/FormWidget.vue";
 import type { PowerState } from "@/libs/xen-api";
 
 const props = defineProps<{
-  state: PowerState | "Unknown";
+  state: PowerState;
 }>();
 
-const icon = computed(() => {
-  const icons = {
-    Running: faPlay,
-    Paused: faPause,
-    Suspended: faMoon,
-    Unknown: faQuestion,
-    Halted: faStop,
-  };
-
-  return icons[props.state];
-});
+const icons = {
```

Comment:
`ICONS_BY_POWER_STATE`?

Reply:
Do we have a naming convention for this?

What would make us choose `UPPERCASE` vs `camelCase` for a `const` (for example, just above we got a `const props`, not a `const PROPS`.

On old JavaScript, `UPPERCASE` was useful to differentiate `var iAmAVariable` and `var I_AM_A_CONSTANT`, but is it still pertinent today?

What's your thought on that?